cime/scripts/lib/CIME/SystemTests/pgn.py

```diff
@@ -60,7 +60,7 @@
 prtstr     = ['woprt','posprt','negprt']
 
 #file name for file containing PGE cloud
-fcld_nc  = 'cloud.nc'
+fcld_nc  = 'PGN_cloud.nc'
```

Comment:
global variables should have all-caps names.

Reply:
Thanks! I didn't know that. I will make that change in this script.